model/bootstrap/node_info.go

```diff
@@ -220,6 +221,15 @@ func FilterByRole(nodes []NodeInfo, role flow.Role) []NodeInfo {
 	return filtered
 }
 
+// Order sorts the NodeInfo list in place using the given ordering.
+func Order(nodes []NodeInfo, order flow.IdentityOrder) []NodeInfo {
```

Comment:
Maybe refactoring `Order` to `Sort` would reduce the cognitive load of what does it mean:
```
func Sort(nodes []NodeInfo, sorter flow.IdentityOrder) []NodeInfo
```

Reply:
👍  I agree that is clearer. Updated this method as well as the `IdentityList` methods in https://github.com/onflow/flow-go/pull/678/commits/2d834ed5d3da080cef5171006afd40045f5262cd